utilities/cli/docker_config.json

```diff
@@ -0,0 +1,70 @@
+{
+	"modes": {
```

Comment:
Project metadata.json modes are exclusive, but here it seems like we will typically apply most of the docker arg collections below to a given container launch command